utils/vscode/syntaxes/carbon.tmLanguage.json

```diff
@@ -0,0 +1,178 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
```

Comment:
As a mechanical thing, we use two-space indentation rather than tabs, and I think we should apply the same rule to this JSON file. Can you make that change, please?